src/snovault/schema_utils.py

```diff
@@ -281,6 +282,19 @@ def notSubmittable(validator, linkTo, instance, schema):
     yield ValidationError('submission disallowed')
 
 
+# modifying https://github.com/lrowe/jsonschema_serialize_fork/blob/master/jsonschema_serialize_fork/_validators.py#L143
+def pattern(validator, patrn, instance, schema):
+    if isinstance(patrn, list):
```

Comment:
It looks like there is dependency in JS form that expects this to be string? https://github.com/Lattice-Data/snovault/blob/d6bec11aa60244172d52512d918fc65663071073/src/snowflakes/static/components/form.js#L293